website/docs/gator.md

```diff
@@ -62,6 +64,23 @@ Or you can specify both as flags:
 gator test -f=my-manifest.yaml -f=templates-and-constraints/
 ```
 
+The `--image` flag specifies a content addressable OCI artifact containing
+policy files. The image(s) will be copied into the local filesystem in a
+temporary directory, the location of which can be overridden with
+the `--tempdir`
+flag. Only files with the aforementioned extensions will be processed. For
```

Comment:
we say `aforementioned` here but we actually mention the extension below when we talk about in the `Bundling Policy into OCI Artifacts` section for the first time right?

Reply:
see line 50-53: 

>  If a file is specified, that file must end in one of the following extensions: `.json`, `.yaml`, `.yml`.